src/compose.js

```diff
@@ -10,14 +10,15 @@
  */
  
 export default function compose(...funcs) {
+  const isEmpty = funcs.length === 0
```

Comment:
Let’s do it the other way: return another function early if there are no `funs`.

```js
export default function compose(...funcs) {
  if (!funcs.length) {
    return (...args) => args[0];
  }

  // ...

}
```

Then we can remove those ternaries and conditions below.